hack/terraform-quickstart/main.tf

```diff
@@ -25,6 +25,17 @@ resource "aws_instance" "worker_node" {
   }
 }
 
+resource "aws_instance" "master_node" {
+  ami           = "${data.aws_ami.coreos_ami.image_id}"
+  instance_type = "m3.medium"
+  key_name      = "${var.ssh_key}"
+  count         = "${var.additional_masters}"
```

Comment:
Actually "additional masters" is a bit vague. Can we just use a specific number: "master_count" or some such?